### PR TITLE
Initial docs for visualization

### PR DESCRIPTION
**Pull request by tomconte**
The following changes are proposed:

* Initial docs for visualization

### CODE REVIEW

File: docs/visualization/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+- Average
+- Count
+- Max
+- Median
+- Min
+- Percentiles
+- Standard Deviation
+- Sum
+- Unique Count
```
Comment (eladiw) [inline docs/visualization/README.md:38]: Also add: Top Hits (will be merged this week)

File: docs/visualization/filters.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Filters
+
+Filters is a multi-bucket aggregation where each bucket contains the documents that match a query.
+
+**Important note:** K2Bridge only supports the Lucene syntax for queries. Make sure you change the default syntax in the Filters aggregation ("KQL) to "Lucene", using the button next to the search box.
+
+![](images/filters.png)
```
Comment (eladiw) [inline docs/visualization/filters.md:7]: image link broken

File: docs/visualization/date_range.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Date Range
+
+This is a range aggregation that is dedicated for date values. Each date (from and to) can be expressed as a Date Math expression, which allows absolute dates, date operations (addition/subtraction), and rounding. You can learn about this format by clicking on the "Acceptable date formats" link in Kibana.
+
+![](images/date_range.png)
```
Comment (eladiw) [inline docs/visualization/date_range.md:5]: image link broken

**Review by eladiw — CHANGES_REQUESTED** [review]
looks good. some minor comments

**Review by AsafMah — APPROVED** [review]
Looks good overall!

File: docs/visualization/README.md (hunk 1 above)
Comment (tomconte) [reply docs/visualization/README.md:38]: Done

File: docs/visualization/filters.md (hunk 2 above)
Comment (tomconte) [reply docs/visualization/filters.md:7]: Not sure what the problem is? The image is there and looks OK when I open the page in GitHub

**Review by eladiw — APPROVED** [review]
lgtm

**Pull request merged**